packages/malloy/src/lang/test/imports.spec.ts

```diff
@@ -93,7 +93,7 @@ source: botProjQSrc is from(->botProjQ)
       },
     });
     docParse.translate();
-    expect(docParse).not.toBeErrorless();
+    expect(docParse).not.toParse();
```

Comment:
I think we mentioned the possibility of `toParse()` and `toTranslate()` to fail if there were warnings or errors. I *think* that `checkForErrors` in the definition of `toParse()` allows warnings — though maybe this is just something for me to fix in my PR...

Also, `.not.toParse()` would then mean "has warnings or errors." Maybe we should try not to use `.not.toParse()` and instead have a `.parseToFailWith(...)`?

Reply:
yeah that's probably a bad test i'll fix it